Building of a project fails due to profile

I have some settings/modules configured in `$PROFILE` (`terminal-icons`, `posh-git`, `oh-my-posh`).

Running a build fails for me with a bunch of errors:

```text
MSBuild version 17.5.0+6f08c67f3 for .NET
  Determining projects to restore...
  Restored C:\_dev\gitextensions-test\src\GitExtensions.PluginTemplate\GitExtensions.PluginTemplate.csproj (in 293 ms).
  Import-PowerShellDataFile : The term 'Import-PowerShellDataFile' is not recognized as the name of a cmdlet, function,

  script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
  correct and try again.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:210 char:33
  +         $hash.Add($_.Basename, (Import-PowerShellDataFile $_.FullName ...
  +                                 ~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (Import-PowerShellDataFile:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException

  Import-PowerShellDataFile : The term 'Import-PowerShellDataFile' is not recognized as the name of a cmdlet, function,

  script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
  correct and try again.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:196 char:22
  +         $colorData = Import-PowerShellDataFile $_.FullName
  +                      ~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (Import-PowerShellDataFile:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:197 char:9
  +         $hash[$colorData.Name] = $colorData
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:198 char:9
  +         $hash[$colorData.Name].Types.Directories[''] = $colorReset
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:199 char:9
  +         $hash[$colorData.Name].Types.Files['']       = $colorReset
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Import-PowerShellDataFile : The term 'Import-PowerShellDataFile' is not recognized as the name of a cmdlet, function,

  script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
  correct and try again.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:196 char:22
  +         $colorData = Import-PowerShellDataFile $_.FullName
  +                      ~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : ObjectNotFound: (Import-PowerShellDataFile:String) [], CommandNotFoundException
      + FullyQualifiedErrorId : CommandNotFoundException

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:197 char:9
  +         $hash[$colorData.Name] = $colorData
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:198 char:9
  +         $hash[$colorData.Name].Types.Directories[''] = $colorReset
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Index operation failed; the array index evaluated to null.
  At C:\Program Files\WindowsPowerShell\Modules\terminal-icons\0.10.0\Terminal-Icons.psm1:199 char:9
  +         $hash[$colorData.Name].Types.Files['']       = $colorReset
  +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
      + FullyQualifiedErrorId : NullArrayIndex

  Extraction root path is 'C:\_dev\gitextensions.shared'.
  Searching for Git Extensions release 'latest' on 'GitHub'.
  Selected release 'v4.0.2'.
  Selected asset 'GitExtensions-Portable-4.0.2.16100-25100ec1f.zip'.
  Downloading 'https://github.com/gitextensions/gitextensions/releases/download/v4.0.2/GitExtensions-Portable-4.0.2.161
  00-25100ec1f.zip'...
  Application extracted to 'C:\_dev\gitextensions.shared'.
C:\Program Files\dotnet\sdk\7.0.202\Microsoft.Common.CurrentVersion.targets(3312,5): error MSB3823: Non-string resource
s require the property GenerateResourceUsePreserializedResources to be set to true. [C:\_dev\gitextensions-test\src\Git
Extensions.PluginTemplate\GitExtensions.PluginTemplate.csproj]
C:\Program Files\dotnet\sdk\7.0.202\Microsoft.Common.CurrentVersion.targets(3312,5): error MSB3822: Non-string resource
s require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [C:\_
dev\gitextensions-test\src\GitExtensions.PluginTemplate\GitExtensions.PluginTemplate.csproj]

Build FAILED.

C:\Program Files\dotnet\sdk\7.0.202\Microsoft.Common.CurrentVersion.targets(3312,5): error MSB3823: Non-string resource
s require the property GenerateResourceUsePreserializedResources to be set to true. [C:\_dev\gitextensions-test\src\Git
Extensions.PluginTemplate\GitExtensions.PluginTemplate.csproj]
C:\Program Files\dotnet\sdk\7.0.202\Microsoft.Common.CurrentVersion.targets(3312,5): error MSB3822: Non-string resource
s require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references. [C:\_
dev\gitextensions-test\src\GitExtensions.PluginTemplate\GitExtensions.PluginTemplate.csproj]
    0 Warning(s)
    2 Error(s)
```

My guess is, that applying `-NoProfile` to the call to PowerShell should fix this and at the same time, it will not have negative impacts.

--- a/src/GitExtensions.Extensibility/build/GitExtensions.Extensibility.targets
+++ b/src/GitExtensions.Extensibility/build/GitExtensions.Extensibility.targets
@@ -31,7 +31,7 @@
         <Message Text="Executing prebuild event provided by GitExtensions.Extensibility" />
         <MakeDir Directories="$(_GitExtensionsDownloadPath)" />
         <Error Condition="!Exists($(GitExtensionsExecutablePath)) and !Exists($(_GitExtensionsDownloadScriptPath))" Text="Path to Git Extensions portable download script is wrong. Current value '$(_GitExtensionsDownloadScriptPath)'." />
-        <Exec Condition="!Exists($(GitExtensionsExecutablePath))" Command="powershell.exe -ExecutionPolicy Unrestricted $(_GitExtensionsDownloadScriptPath) -ExtractRootPath $(_GitExtensionsDownloadPath) -Version $(GitExtensionsReferenceVersion) -Source $(GitExtensionsReferenceSource)" />
+        <Exec Condition="!Exists($(GitExtensionsExecutablePath))" Command="powershell.exe -NoProfile -ExecutionPolicy Unrestricted $(_GitExtensionsDownloadScriptPath) -ExtractRootPath $(_GitExtensionsDownloadPath) -Version $(GitExtensionsReferenceVersion) -Source $(GitExtensionsReferenceSource)" />
         <Message Text="Completed prebuild event provided by GitExtensions.Extensibility" />
     </Target>
 </Project>

